app/Http/Controllers/AuthController.php

```diff
@@ -64,7 +77,12 @@ public function login(Request $request)
             return response()->json(['error' => 'Unauthorized'], 401);
         }
 
-        return $this->respondWithToken($token);
+        return response()->json(
+            array_merge(
+                ['user' => (new UserResource(User::find(auth('api')->user()->id)))->toArray($request)],
```

Comment:
I think we should go back to having a username. In that case, this UserResource will need to return a username